app/screens/more_dms/more_dms.js

```diff
@@ -96,13 +97,21 @@ class MoreDirectMessages extends PureComponent {
             this.setState({profiles, showNoResults: true});
         } else if (this.state.searching &&
             nextProps.searchRequest.status === RequestStatus.SUCCESS) {
-            let results = filterProfilesMatchingTerm(nextProps.profiles, this.state.term);
+            const exactMatches = [];
```

Comment:
This isn't new behaviour here, but because this is triggering on every `componentWillReceiveProps` without checking if there has been a change in props, it's changing the state every single time this is called, so it's likely having a performance impact. We should remember this if we're ever investigating the performance of this screen while searching